firecrest/cli2/__init__.py

```
@@ -0,0 +1,997 @@
+    "repr.attrib_value": "none",
+    "repr.number": "none",
+}
+console = Console(theme=Theme(custom_theme))
+client: fc.v2.Firecrest = None  # type: ignore
+logger = logging.getLogger(__name__)
+
+
+def examine_exeption(e: Exception) -> None:
+    msg = f"{__app_name__}: Operation failed"
+    if isinstance(e, fc.ClientsCredentialsException):
+        msg += ": could not fetch token"
+    elif isinstance(e, fc.FirecrestException):
+        msg += ": a Firecrest client error has occurred"
+    # else:
+        # in case of FirecrestException and ClientsCredentialsException
+        # we don't need to log again the exception
+    msg += f": {e}"
+
+    console.print(f"[red]{msg}[/red]")
+
+
+def version_callback(value: bool):
+    if value:
+        console.print(f"FirecREST CLI Version: {__version__}")
+        raise typer.Exit()
+
+
+def config_parent_load_callback(ctx: typer.Context, param: typer.CallbackParam, value: str):
+    ctx.default_map = ctx.parent.default_map  # type: ignore
+
+
+def config_callback(ctx: typer.Context, param: typer.CallbackParam, value: str):
+    if value:
+        try:
+            with open(value, 'r') as f:
+                config = yaml.safe_load(f)
+
+            ctx.default_map = ctx.default_map or {}
+            ctx.default_map.update(config)
+        except Exception as ex:
+            raise typer.BadParameter(str(ex))
+
+    return value
+
+
+@app.command(rich_help_panel="Status commands")
+def systems(
+    name: Optional[str] = typer.Option(
+        None, "-n", "--name", help="Get information for only one system."
+    ),
+    raw: bool = typer.Option(False, "--json", "--raw", help="Print the output in json format."),
+    pager: Optional[bool] = typer.Option(
+        True, help="Display the output in a pager application."
+    ),
+):
+    """Provides information for the available systems in FirecREST"""
+    try:
+        if name:
+            raise NotImplementedError("Rich table not implemented yet")
```

Maybe this should raise the error with a different message?